### PR TITLE
fix(s3): regions

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: storage/object/quickstart.mdx [hunk 1]
```diff
@@ -53,8 +53,6 @@ To get started with Object Storage, you must first create a bucket. Objects are
       - Region: `nl-ams`
     - Paris, France: 
       - Region: `fr-par`
-    - Warsaw, Poland: 
-      - Region: `pl-waw`
```
Comment (qselle) [inline storage/object/quickstart.mdx:57]: Object Storage is available in pl-waw. 😄

Comment (qselle) [reply storage/object/quickstart.mdx:57]: It has to be specified here: https://github.com/scaleway/docs-content/pull/890/files#diff-fbcd41286715e50712518a5263d0e38c47971a609352df184fcf439126a903a4R92 (Glacier only available in `fr-par` and `nl-ams`)

Comment (ldecarvalho-doc) [reply storage/object/quickstart.mdx]: ah yeah, sorry. my bad! it's fixed

**Review by RoRoJ — APPROVED** [review]

**Comment by qselle** [comment]
LGTM :+1:

**Review by nedaehm — APPROVED** [review]

**Pull request merged**